### ISSUE TITLE
Crackpack outdated mods

### ISSUE THREAD

**Issue by Drizna** [sciguyryan/TheCrackPack#881]
Hey guys, I am playing on a friends server, and I was looking through the mod list. While browsing I realized that majority of the mods on here have a newer version out, and probably will fix a lot of the issues on the server. If possible could the mods be updated to their newer versions to fix these issues? Also it says 188 out of 189 are running, and I am confused as to which mod is on the list, but not working. I also noticed certain things that kill the server when activated. Auto-spawner, and the osmotic enchanter so far. But also the essence refiner when activated to make an essence kills the refiner itself to the point that no one in survival can remove the block, not even an op in creative could. It took tnt to cause the block to be removed. I would list all the issues I have seen so far here, but I don't want to take up a lot of space.

**Comment by Drizna** [comment]
Heck I even noticed that the witches don't have a dying noise, and it comes up as an issue on the ATL console log.

**Comment by evknucklehead** [comment]
It takes a lot of work to keep a pack up to date, especially with some mods updating 2-3 times per week or more frequently. Yes, they are aware of the issues that are fixed in the newer mod versions, but it's not a simple process to keep all the mods playing nicely with each other. Sometimes they also wait for something major (ie commonly used by players) to be updated before they work on updating the rest.

Add in the fact that mods are starting to make the transition to MC 1.8, and it becomes even more of a dilemma: Do they release a pack update that will be obsolete in a week or two, or wait until the mods are ready for 1.8? 

As for the witches' dying noise, Vanilla Minecraft doesn't have one for them either, so you'll get that *warning* on a vanilla install through the launcher. Not all warnings (or "errors", for that matter) are actual problems. It usually means something is not done in quite the way MC, Forge, or one of the mods in the pack is expecting. 

Keep in mind that I am just a guy who likes helping people solve their problems, and not one of the pack authors. I don't claim to know everything that goes into their decision making process, just observations from how the pack has been handled so far. 

Bottom line, the pack will be updated when and if the authors feel the mix of mods is stable and worth releasing.
